pkgs/applications/networking/testssl/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, pkgs, lib }:
+
+stdenv.mkDerivation rec {
+  version = "2.9.5-1";
+  name = "testssl.sh-${version}";
+
+  src = fetchFromGitHub {
+    owner = "drwetter";
+    repo = "testssl.sh";
+    rev = "ed88add923139e0c8cf4d9cf59d6f9e6cf5326cf";
+    sha256 = "0hz6g685jwl0c0jrdca746425xpwiwc8lnlc2gigga5hkcq8qzl9";
+  };
+
+  buildInputs = with pkgs; [
+    makeWrapper
+  ];
+
+  # Replace /bin/pwd with the actual path to pwd in coreutils
+  pwdBinPath = "${lib.makeBinPath (with pkgs; [ coreutils ])}/pwd";
+  postPatch = ''
+    sed -i -e "s|/bin/pwd|${pwdBinPath}|g" testssl.sh
+  '';
+
+  installPhase = ''
+    # Make directories
+    mkdir -p $out/bin $out/etc
+
+    # Copy etc folder
+    cp -r etc/ $out/
+
+    # Copy script
+    cp testssl.sh $out/bin/testssl.sh
+
+    # Make wrapper to let the script know where the etc folder is
+    wrapProgram $out/bin/testssl.sh                                            \
+      --set TESTSSL_INSTALL_DIR $out                                           \
+      --set PATH '${lib.makeBinPath (with pkgs; [ openssl ])}:$PATH'
+  '';
+
+  meta = with stdenv.lib; {
+    description = "testssl.sh is a commandline tool to check a server's TLS/SSL capabilities";
+    longDescription = ''
+      testssl.sh is a free command line tool which checks a server's service on
+      any port for the support of TLS/SSL ciphers, protocols as well as recent
+      cryptographic flaws and more.
+    '';
+    homepage = https://testssl.sh/;
+    license = licenses.gpl2;
+  };
```

Comment:
A maintainer must be set, did you want to maintain this package?

Reply:
Sure, I can do that. I will amend and push that together with the version fix.